electron-app/src/server/websites/e621/e621.service.ts

```diff
@@ -165,15 +180,96 @@ export class e621 extends Website {
     context.ifYouWantToCreateNotice = false;
   }
 
+  // Used to debounce long running validation requests
+  private validateSubmissionCancelTokens = new Map<string, CancellationTokenDynamic>();
```

Comment:
checking big amount of tags with the newly implemented rate limit was too slow because postybirb had to process all validation requests, so i decided to implement debounce and cancel old validation requests